cimpl/ecqm_measure_ep.txt

```diff
@@ -616,6 +620,8 @@ Description: "Diabetic Retinopathy: Communication with the Physician Managing On
     0..* LevelOfSeverityOfRetinopathyFindingsCommunicationFromProviderToProvider
     0..* MacularEdemaFindingsAbsentCommunicationFromProviderToProvider
     0..* MacularEdemaFindingsPresentCommunicationFromProviderToProvider
+    0..* MedicalReason1NegationRationale
```

Comment:
Will accept, but for future modeling discussions, but we probably want a better way to specify versions of terminology bindings to the same element other than renaming it "MedicalReason1NegationRationale."  
In analyzing both value sets for MedicalReasonNegationRationale and Medical1ReasonNegationRationale, the only difference is that there is 1) a new vocabulary steward, and 2) more updated terms are added.  With that the element name is the same in definition but is being overloaded with an updated list.